pkgs/development/python-modules/dnspython/default.nix

```diff
@@ -56,10 +55,8 @@ buildPythonPackage rec {
 
   checkInputs = [ cacert ] ++ optional-dependencies.DNSSEC;
 
-  # don't run live tests
-  env = lib.optionalAttrs stdenv.hostPlatform.isDarwin {
-    NO_INTERNET = 1;
-  };
+  # don't run tests that depend on the internet
+  env.NO_INTERNET = 1;
```

Comment:
Looks like that the `preCheck` section can be removed.

Reply:
thanks. I just reverted 4bd2716b3a6eeb3e0cecd5ee6e742ee262dd179c since staging already sets the variable.